enclave_wrangler/dataset_upload.py

```diff
@@ -34,11 +35,131 @@
+
+    # Upload to enclave
+    for linked_cset in linked_csets.values():
+        container_d = linked_cset['container']
+        response = upload_concept_set(  # concept_set_container
+            concept_set_id=container_d['concept_set_name'],
+            intention=container_d['intention'],
+            # todo: research_project: (a) default this to ENCLAVE_PROJECT_NAME in func, (b) do that here, (c) add it as a column to an
+            #  updated palantir-3-file for the new api
+            research_project=ENCLAVE_PROJECT_NAME,
+            assigned_sme=PALANTIR_ENCLAVE_USER_ID_1,
+            assigned_informatician=PALANTIR_ENCLAVE_USER_ID_1,
+            # TODO: validate first (if this func says validate_first=True, then upload
+            validate=True)
+
+        for version in linked_cset['versions'].values():
+            version_d = version['version']
+            response = upload_draft_concept_set(  # code_set
+                # todo: domain_team: @Siggie: Not sure what to put here, but it is optional param, so I'm leaving blank.
+                # domain_team=version_d[''],
+                provenance=version_d['provenance'],
+                # todo: current-max-version: @Siggie: Sucks that we have to set this. but probaby because version types
+                #  can be arbitrary, so enclave can't know. but this means we have to have a local registry for most
+                #  recent version as well for anything we're uploading, or get it when we re-download, right?
+                #  ...docs, for reference: This must be set to the current maximum version number assigned to a version
+                #  of this concept set, or null if creating the first version of a concept set. If null, then
+                #  baseVersion is not "required".
+                #  ...Leaving this blank for now since is optional param.
+                # current_max_version=version_d[''],
+                # todo: concept_set: is this correct?
+                concept_set=version_d['concept_set_name'],  # == container_d['concept_set_name']
+                # todo: annotation: this should be moved into the new palantir-3-file data model, whatever that is
+                annotation='Curated value set: ' + version_d['concept_set_name'],
+                limitations=version_d['limitations'],
+                intention=version_d['intention'],
+                # todo: base_version: @Siggie: Leaving out, since is optional param, and not possible to include without
+                #  first adding current_max_version, for which more time is needed to figure out how to reliably ascertain.
+                # base_version=1,
+                # todo: intended_research_project: (a) default this to ENCLAVE_PROJECT_NAME in func, (b) do that here, (c) add it as a column to an
+                #  updated palantir-3-file for the new api
+                intended_research_project=ENCLAVE_PROJECT_NAME,
```

Comment:
I'm assuming that the concept set version `intended_research_project` is the same as the container `research_project`, which we default to `RP-4A9E27`.